packages/augur-ui/src/modules/reporting/actions/participation-tokens-management.ts

```diff
@@ -26,6 +29,11 @@ export const purchaseParticipationTokens = (
     );
     return callback(null, gas);
   }
-  buyParticipationTokens(universeId, amount);
+  dispatch(addPendingData(BUY_PARTICIPATION_TOKENS, BUY_PARTICIPATION_TOKENS, TXEventName.Pending, BUY_PARTICIPATION_TOKENS));
+  buyParticipationTokens(universeId, amount).then(() => {
+    dispatch(removePendingData(BUY_PARTICIPATION_TOKENS, BUY_PARTICIPATION_TOKENS))
```

Comment:
don't use `then` to remove pending queue use log-handler ``handleParticipationTokensRedeemedLog